packages/plugin-core/package.json

```diff
@@ -64,18 +64,12 @@
           "contextualTitle": "Sample View",
           "type": "webview",
           "when": "dendron:devMode"
-        }
-      ],
-      "dendron-view": [
-        {
-          "id": "dendron.tip-of-the-day",
-          "name": "Tip of the Day",
-          "when": "dendron:pluginActive",
-          "type": "webview"
         },
         {
-          "id": "dendron.backlinks",
-          "name": "Backlinks",
+          "id": "dendron.calendar-view",
```

Comment:
heads up that this should be in the `dendron-view`, not in the `explorer`. currently, `gen:config` doesn't account for this so it will need to be updated

Reply:
`gen:config` is generating them correctly now after merging master. I had to correct some more things but I think it's good now.